util/Xamarin.AndroidBinderator/Xamarin.AndroidBinderator/BindingProjectConverter.cs

```diff
@@ -27,6 +27,7 @@ public static List<BindingProjectModel> Convert (BindingConfig config)
 	// Converts an artifact from config.json to the model used to generate files
 	static BindingProjectModel ConvertArtifact (BindingConfig config, MavenArtifactConfig mavenArtifact, List<Exception> exceptions)
 	{
+		Console.WriteLine($"{mavenArtifact.GroupAndArtifactId}:{mavenArtifact.Version}");
```

Comment:
There should be a space before the opening parentheses here.  We have an `.editorconfig` file in this repo that should format this correctly.  You might want to make sure something isn't messed up locally with your IDE:

https://github.com/dotnet/android-libraries/blob/main/util/Xamarin.AndroidBinderator/.editorconfig#L193